ironfish/src/wallet/exporter/accountImport.ts

```diff
@@ -44,3 +55,54 @@ export function toAccountImport(account: Account, viewOnly: boolean): AccountImp
 
   return value
 }
+
+export function validateAccountImport(toImport: AccountImport): void {
```

Comment:
This was the old `validators/validateAccount` function. However, It's only used to validate an AccountImport and so I moved it in here. There's no need to ever validate an account because the import is verified before an account is constructed. So you only want to verify an AccountImport.